tests/components/rfxtrx/test_cover.py

```diff
@@ -140,3 +140,180 @@ async def test_duplicate_cover(hass, rfxtrx):
     assert state
     assert state.state == "closed"
     assert state.attributes.get("friendly_name") == "LightwaveRF, Siemens 0213c7:242"
+
+
+async def test_rfy_cover(hass, rfxtrx):
+    """Test Rfy venetian blind covers."""
+    entry_data = create_rfx_test_cfg(
+        devices={
+            "071a000001020301": {
+                "signal_repetitions": 1,
+                "venetian_blind_mode": "Unknown",
+            },
+            "071a000001020302": {"signal_repetitions": 1, "venetian_blind_mode": "US"},
+            "071a000001020303": {"signal_repetitions": 1, "venetian_blind_mode": "EU"},
+        }
+    )
+    mock_entry = MockConfigEntry(domain="rfxtrx", unique_id=DOMAIN, data=entry_data)
+
+    mock_entry.add_to_hass(hass)
+
+    await hass.config_entries.async_setup(mock_entry.entry_id)
+    await hass.async_block_till_done()
+
+    # Test a blind with no venetian mode setting
+    state = hass.states.get("cover.rfy_010203_1")
+    assert state
+
+    await hass.services.async_call(
+        "cover",
+        "stop_cover",
+        {"entity_id": "cover.rfy_010203_1"},
+        blocking=True,
+    )
+
+    await hass.services.async_call(
+        "cover",
+        "open_cover",
+        {"entity_id": "cover.rfy_010203_1"},
+        blocking=True,
+    )
+
+    await hass.services.async_call(
+        "cover",
+        "close_cover",
+        {"entity_id": "cover.rfy_010203_1"},
+        blocking=True,
+    )
+
+    await hass.services.async_call(
+        "cover",
+        "open_cover_tilt",
+        {"entity_id": "cover.rfy_010203_1"},
+        blocking=True,
+    )
+
+    await hass.services.async_call(
+        "cover",
+        "close_cover_tilt",
+        {"entity_id": "cover.rfy_010203_1"},
+        blocking=True,
+    )
+
+    assert rfxtrx.transport.send.mock_calls == [
+        call(bytearray(b"\x08\x1a\x00\x00\x01\x02\x03\x01\x00")),
+        call(bytearray(b"\x08\x1a\x00\x01\x01\x02\x03\x01\x01")),
+        call(bytearray(b"\x08\x1a\x00\x02\x01\x02\x03\x01\x03")),
+    ]
```

Comment:
There are only three mock calls here but we called cover services five times. Is this correct?

Reply:
Yes, this is correct.  This is a test for a non-venetian Rfy cover.